data/bridge2ai_voice_project_data_dictionary.csv

```diff
@@ -1173,8 +1173,10 @@ mph_session_id,q_mood_participant_history,"Questionnaire - Metadata",text,"Sessi
 mph_started_at,q_mood_participant_history,,text,"Questionnaire Started At",,,,,,y,,y,,,,,
 mph_completed_at,q_mood_participant_history,,text,"Questionnaire Completed At",,,,,,y,,y,,,,,
 mph_duration,q_mood_participant_history,,text,"Questionnaire Duration (seconds)",,,number,0,,,,,,,,,
-mph_active_psy_med_problems,q_mood_participant_history,"Medical and Psychiatric comorbidities",checkbox,"Please select any active psychiatric problems and medical problems (select all that apply).","adhd, ADHD | anxiety, Anxiety | ocd, OCD | stroke, Stroke | epilepsy, Epilepsy | laryngeal_cancer, Laryngeal cancer | seasonal_allergies, Seasonal allergies | other, Other",,,,,,,,,,,,
+mph_active_psy_medical_problems,q_mood_participant_history,"Medical and Psychiatric comorbidities",text,"Please list any active psychiatric problems and medical problems.",,,,,,,,,,,,,
+mph_active_psy_med_problems,q_mood_participant_history,,checkbox,"Please select any active psychiatric problems and medical problems (select all that apply).","adhd, ADHD | anxiety, Anxiety | ocd, OCD | stroke, Stroke | epilepsy, Epilepsy | laryngeal_cancer, Laryngeal cancer | seasonal_allergies, Seasonal allergies | other, Other",,,,,,,,,,,,
 mph_active_psy_med_prob_other,q_mood_participant_history,,text,"Please specify",,,,,,,"[mph_active_psy_med_problems(other)]=""1""",,,,,,
+mph_hist_psy_medical_problems,q_mood_participant_history,,text,"Please list any historical psychiatric problems and medical problems (select all that apply).",,,,,,,,,,,,,
```

Comment:
here there should not be a select all that apply

Reply:
Thanks. Will update right away